engine/apps/slack/scenarios/shift_swap_requests.py

```diff
@@ -186,8 +186,11 @@ def post_message_to_thread(
         if not shift_swap_request.slack_message:
             return
 
+        # TODO: once _channel_id has been fully migrated to channel, remove _channel_id
+        # see https://raintank-corp.slack.com/archives/C06K1MQ07GS/p173255546
         self._slack_client.chat_postMessage(
-            channel=shift_swap_request.slack_message.channel.slack_id,
+            # channel=shift_swap_request.slack_message.channel.slack_id,
+            channel=shift_swap_request.slack_message._channel_id,
```

Comment:
"fix" from https://github.com/grafana/oncall/pull/5292 (let's wait to read from channel until the data-migration mentioned in that PR has been fully finished)